.github/workflows/pre-commit.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     uses: observeinc/.github/.github/workflows/shared_commit-validation.yaml@main
     secrets: inherit
     with:
-      skip: '{}'
+      skip: '{"jobs": ["single-commit"]}'
```

Comment:
PLEASE remove this skip. CI workflows depend on a single commit being present. Otherwise we CANNOT determine the nature of commit to version the chart!